creating_packages/getting_started.rst

```diff
@@ -4,17 +4,17 @@ Getting Started
 ===============
 
 To start learning about creating packages, we will create a package from the existing source code
-repository: https://github.com/memsharded/hello. You can check that project, it is a very simple
+repository: https://github.com/conan-io/hello. You can check that project, it is a very simple
 "hello world" C++ library, using CMake as the build system to build a library and an executable. It does not contain
 any association with Conan.
 
 We are using a similar GitHub repository as an example, but the same process also applies to other source
-code origins, like downloading a zip or tarball from the internet.
+code origins, like downloading a zip or tar ball from the internet.
```

Comment:
This was ok, ``tarball`` is used in computing: https://en.wikipedia.org/wiki/Tar_(computing)